rswitcher@nixdev.com/files/rswitcher@nixdev.com/rswitcher

```diff
@@ -129,3 +129,6 @@ fi
 
 # Run the xrandr function to create the new settings.
 create_resolution
+
+# Reload start menu to get correct dimentions
+dbus-send --session --dest=org.Cinnamon.LookingGlass --type=method_call /org/Cinnamon/LookingGlass org.Cinnamon.LookingGlass.ReloadExtension string:'menu@cinnamon.org' string:'APPLET'
```

Comment:
You can also use /org/Cinnamon and org.Cinnamon and the ReloadXlet method call. I think that is the proper scope but this looks like it would still work too.

Reply:
Thanks for the feedback! I will check it out. :+1: